server/subscribe.go

```diff
@@ -116,13 +116,9 @@ func (p *Plugin) getChannelsSubscribed(wh *webhook) ([]string, error) {
 		if eventTypes.Intersection(webhookEvents).Len() > 0 {
 			foundEvent = true
 		} else if eventTypes.ContainsAny(eventUpdatedAny) {
-			if webhookEvents.Intersection(updateEvents).Len() > 0 {
```

Comment:
Why was this taken out? (I can't tell just from the code)

Reply:
The line removed is asserting that the incoming event matches up with one of the events defined for updating a built-in field. Having the prefix `event_updated_` lets us know that we are updating some field, whether it is built-in or custom. Checking that it is one of the default fields would be redundant so I removed that check.